### PR TITLE
FIX Last Visit date not updated after user logged in

### PR DESCRIPTION
**Pull request by ITernovtsii**
We have developed custom cronjob to disable user who not logged in after some time after creating (by checking `ezuservisit` table).
But, Last Visit date not updated if user logged in providing old and new passwords in this module (userpaex/password).
I believe it should be updated in the same way as in standard "user/login" module.

### CODE REVIEW

File: modules/userpaex/password.php [hunk 1]
```diff
@@ -110,6 +110,7 @@
                         $oldPassword = '';
 
                         eZUser::setCurrentlyLoggedInUser( $user, $UserID );
+                        eZUser::updateLastVisit( $UserID );
```
Comment (glye) [inline modules/userpaex/password.php]: From the description it looks like this module logs you in. In that case it should probably update the login counter, too. Meaning, the second parameter should be set to true. If login doesn't happen here, then never mind.
https://github.com/ezsystems/ezpublish-legacy/blob/0d8aebb911ec2d8d4665c214797ae7629f134d7e/kernel/classes/datatypes/ezuser/ezuser.php#L1528
```suggestion
                        eZUser::updateLastVisit( $UserID, true );
```

**Review by glye — APPROVED** [review]

**Comment by glye** [comment]
@andrerom I don't have any permissions on this repo, can you tag/review/merge please?

**Review by andrerom — APPROVED** [review]
+1

**Comment by glye** [comment]
Thanks, both of you! 🎉

**Pull request merged**